.github/pull_request_template.md

```diff
@@ -4,9 +4,17 @@
 
 *Add screenshots or screen recordings for visual / UI-focused changes.*
 
+
 ## Checklist
-- [ ] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
-- [ ] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
-- [ ] Words are spelled using American English
-- [ ] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)
 - [ ] If I moved a page, I added a redirect in `vercel.json`
+- [ ] I've added (at least) 3 to 5 internal links to this new article
+- [ ] I've added keywords for this page to the rank tracker in Ahrefs
```

Comment:
Just tried this for the first time. How many variations of the keyword should I add?
e.g. I just wrote a tutorial on AB testing on iOS. Should I just add 'iOS ab testing" to the rank tracker? Or should I also add things like "ab testing ios", "iOS app ab testing", "how to ab test on iOS", etc

Reply:
Hilariously, I just tested this and we rank several positions lower for the singular than the plural. Madness.